custom_components/google_home/api.py

```diff
@@ -149,6 +151,7 @@ async def collect_data_from_endpoints(
     ) -> GoogleHomeDevice:
         """Collect data from different endpoints."""
         device = await self.update_alarms_and_timers(device)
+        device = await self.update_alarm_volume(device)
```

Comment:
Yes, this is required to keep the value in sync if it's changed from the Google Home app or by voice command.